packages/amplify-e2e-core/src/init/deleteProject.ts

```diff
@@ -23,3 +23,18 @@ export const deleteProject = async (cwd: string, profileConfig?: any, usingLates
     .wait('Project deleted locally.')
     .runAsync();
 };
+
+export const amplifyDeleteWithLongerTimeout = (cwd: string, usingLatestCodebase = false): Promise<void> => new Promise((resolve, reject) => {
```

Comment:
Can we override the existing method instead of creating a new one?

Reply:
+1 on this